presto-kafka/src/main/java/io/prestosql/plugin/kafka/encoder/json/format/MillisecondsFormatFunctions.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.kafka.encoder.json.format;
+
+import java.util.function.Function;
+
+import static io.prestosql.spi.type.DateTimeEncoding.unpackMillisUtc;
+
+public final class MillisecondsFormatFunctions
+{
+    private MillisecondsFormatFunctions() {}
+
+    public static Function<Long, String> formatMillisFunc()
+    {
+        return String::valueOf;
+    }
+
+    public static Function<Long, String> formatMillisWithTZFunc()
+    {
+        return encodedMillisWithTZ -> String.valueOf(unpackMillisUtc(encodedMillisWithTZ));
```

Comment:
Perhaps we should not support this scenario.  This does not seem a meaningful return value - without the tz info, the milliseconds don't make much sense here, right?

Reply:
Sorry, that was wrong.  This format implies millis since epoch.